modin/pandas/__init__.py

```diff
@@ -113,7 +113,7 @@ def _update_engine(publisher: Parameter):
     elif publisher.get() == "Native":
         # With OmniSci backend there is only a single worker per node
         # and we allow it to work on all cores.
-        if Backend.get() == "OmniSci":
+        if Backend.get() == "omnisci":
```

Comment:
```suggestion
        if Backend.get() == "Omnisci":
```
IIRC `Backend.get()` capitalizes the string